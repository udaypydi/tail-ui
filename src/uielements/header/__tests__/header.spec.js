import React from 'react';
import { render } from 'test-utils';
import Header from 'uielements/header/header.component.js';

it('should take a snapshot', () => {
    const { asFragment } = render(<Header />)
    
    expect(asFragment(<Header />)).toMatchSnapshot()
});
