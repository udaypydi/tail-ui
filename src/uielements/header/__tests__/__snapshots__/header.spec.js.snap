// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <p
    class="font-bold text-6xl"
  />
</DocumentFragment>
`;
